adminmain.php

```diff
@@ -66,19 +45,41 @@
             </div>
             </div>
     <?php endif ?>
+</head>
+<body style="overflow: hidden;">
+    <div id="main-content"  style="overflow-x: hidden; overflow-y: scroll; height: 89%;"></div>
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"
+    integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+    crossorigin="anonymous"></script>
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" 
     integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" 
     crossorigin="anonymous"></script>
+    <script src="changepage.js"></script>
```

Comment:
jQuery shouldn't be placed within the `<head>` tag, reference: https://learn.jquery.com/about-jquery/how-jquery-works/ . Since you're using Bootstrap too (it uses jQuery), jQuery needs to be called before BootstrapJS, and the BootstrapCSS doesn't use jQuery so it stays on top within the `<head>` tag.

Order: jQuery > Bootstrap(JS) > other scripts